dwave/system/testing.py

```diff
@@ -97,6 +97,47 @@ def sample(self, bqm, num_reads=10, flux_biases=[], **kwargs):
                                                  for sample in response.samples()],
                                                 bqm, info=info)
 
+class MockLeapHybridDQMSampler:
+    """Mock sampler modeled after LeapHybridDQMSampler that can be used for tests."""
+    def __init__(self, **config):
+        self.parameters = {'time_limit': ['parameters'],
+                           'label': []}
+
+        self.properties = {'category': 'hybrid',
+                           'supported_problem_types': ['dqm'],
+                           'quota_conversion_rate': 20,
+                           'minimum_time_limit': [[20000, 5.0], 
```

Comment:
Was not sure which properties were worth adding to the mock sampler: kept all of them except properties['parameters']

Reply:
FWIW, I'm adding solver data mocks for different solver types in [`dwave.cloud.testing.mocks`](https://github.com/dwavesystems/dwave-cloud-client/pull/463/files#diff-e98c5d73eb0f63bdfd12ce31bb92890e2bb082b3287defbba9c31be06031a499R188). The next step is to add mock Solvers, and then they can be used by mock Samplers here.